apis/v1alpha1/generator.yaml

```diff
@@ -1,11 +1,51 @@
 ignore:
   resource_names:
-    - Snapshot
     - User
     - ACL
     - Cluster
     - SubnetGroup
+  field_paths:
+    - DescribeSnapshotsInput.ClusterName
+    - DescribeSnapshotsInput.Source
 resources:
+  Snapshot:
+    exceptions:
+      errors:
+        404:
+          code: SnapshotNotFoundFault
+      terminal_codes:
+        -  InvalidParameterCombinationException
+        -  InvalidSnapshotStateFault
+        -  SnapshotAlreadyExistsFault
+        -  SnapshotQuotaExceededFault
+        -  TagQuotaPerResourceExceeded
+        -  ClusterNotFoundFault
+        -  InvalidClusterStateFault
+        -  InvalidParameterValueException
+        -  InvalidParameter
+    fields:
+      SourceSnapshotName:
+        from:
+          operation: CopySnapshot
+          path: SourceSnapshotName
+      ClusterName:
```

Comment:
Is this field not automatically generated? Why do we have to add it to the list of fields?

Reply:
This is part of different API (copy-snapshot) so we would need to add it